config/activator.yaml

```diff
@@ -74,6 +74,10 @@ spec:
           mountPath: /etc/config-logging
         - name: config-observability
           mountPath: /etc/config-observability
+        securityContext:
+          runAsUser: 1337
```

Comment:
do these uids/gids need to exist on the worker nodes?

Reply:
Specifying UID and GID is probably not the direction that we want to go in. For example, if you're using pod security policy (see https://kubernetes.io/docs/concepts/policy/pod-security-policy/) or OpenShift security context constraints (the precursor to PSP in the upstream), setting these fields explicitly will likely require use of a security policy of some kind.

I am fine with setting `allowPrivilegeEscalation` to false as has been done here.

For `runAsUser` and `runAsGroup`, is it not sufficient to a non-root UID in the image metadata? And avoid setting these fields here?

Reply:
Using :nonroot distroless image now.